mama/dotnet/src/cs/MamaMsg.cs

```diff
@@ -4870,8 +4871,8 @@ public bool tryStringAnsi(string name, ushort fid, ref IntPtr result)
 			bool throwOnError)
 		{
 			EnsurePeerCreated();
-			ulong dateTimeMsec = 0;
-			int code = NativeMethods.mamaMsg_getDateTimeMSec(nativeHandle, name, fid, ref dateTimeMsec);
+			double dateTimeSec = 0;
+			int code = NativeMethods.mamaMsg_getDateTimeSeconds(nativeHandle, name, fid, ref dateTimeSec);
```

Comment:
I stand corrected - creating a timespec in C# native would be tricky and dangerous - so let's keep this method.